### PR TITLE
[DOCS] Fix `operation_mode` response property def

### PR DESCRIPTION
**Pull request by stefnestor**
I'm not reproducing returning `STARTED` on 7.12.0 nor finding it in the code. 

 I've updated docs to line up to [this code](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/OperationMode.java) (`STARTED` becomes `RUNNING` & added descriptions). 

Kindly verify before accepting.

### Preview
https://elasticsearch_73976.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ilm-get-status.html#ilm-get-status-response-body

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
Thanks @stefnestor. I reformatted this to better align it with some of our other API docs.

This LGTM, but I'd like to get approval from an engineer before merging.

**Review by dakrone — APPROVED** [review]
LGTM, thanks for adding this.

**Comment by jrodewig** [comment]
Thanks @dakrone.

**Pull request merged**
